emartech/3rd-party-integrations-magento-M2
# Extension 1.0.6 version release

Enhancements:
-------------
A. Product Full Catalog Export:
    1. Use API Post Method to export the product catalog to Emarsys
    2. Provide a configuration for the Below fields under Catalog export settings
	a. Export Method (API POST / SFTP)
	b. Merchant ID
	c. Bearer Token
	d. Test Connection [Button]
    3. Make Sure the API Post works aside of FTP feature
    4. When the "Test Conncetion" button clicked push a CSV with only headers & one sample product and run the test

B. Sales Data (SI) Export:
    1. Use API Post Method to export the Orders to Emarsys
    2. Configuration for the Below fields under Catalog export settings
	a. Export Method (API POST / SFTP)
	b. Merchant ID
	c. Bearer Token
	d. Maximum Number of records per export
	e. Test Connection
    3. Make Sure the API Post works aside of FTP feature
    4. When the "Test Conncetion" button clicked push a CSV with only headers & one sample order and run the test

C. Contact (Customer / Subscriber) Export:
    1. Contact Sync using API in both Manual Export (If count of records lessthan 10k) & Batch Export via CRON JOB
    2. Clear the chunk of Customer records from Queue as soon as synced to Emarsys
    3. Clear the chunk of Subscriber records from Queue as soon as synced to Emarsys
    4. Convert the WebDAV script to use Magento core function instead of thirdparty library dependencies
    5. Contact Bulk / Initial Exports via CRON

D. Optin Enhancements:
    1. Subscribe from checkout
    2. Fixed issue with subscriber confirmation email when Double Optin enabled
